### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by HamzaIbnFarooq**
#### What are the relevant tickets?
https://github.com/mitodl/edx-sysadmin/issues/18

#### What's this PR do?
Updates Readme

#### How should this be manually tested?
Follow the Readme to setup the plugin.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,96 +1,90 @@
...
+Earlier, ``Sysadmin Dashboard`` was a part of ``edx-platform``, however starting from `lilac release <https://github.com/edx/edx-platform/tree/open-release/lilac.master>`_ of Open edX the sysadmin panel has been removed
+and transitioned to as separate plugin.
 
-The ``README.rst`` file should start with a brief description of the repository,
-which sets it in the context of other repositories under the ``edx``
-organization. It should make clear where this fits in to the overall edX
-codebase.
+NOTE:
+It is recommended that you use edx-sysadmin plugin with Open edX's `lilac <https://github.com/edx/edx-platform/tree/open-release/lilac.master>`_ release and successors.
+If you wish to use the ``Sysadmin Dashboard`` with Open edX releases before ``lilac`` you should just enable ``ENABLE_SYSADMIN_DASHBOARD`` feature flag in environment files (e.g ``lms.yml`` or ``lms.env.json``) to access sysadmin dashboard features.
```
Comment (ziafazal) [inline README.rst:10]: @HamzaIbnFarooq I don't think we are not relying on `ENABLE_SYSADMIN_DASHBOARD` feature flag. we just need to install plugin and sys admin should be available.

File: README.rst [hunk 2]
```diff
@@ -1,96 +1,90 @@
...
-
-Overview (please modify)
+Overview
 ------------------------
 
-The ``README.rst`` file should then provide an overview of the code in this
-repository, including the main components and useful entry points for starting
-to understand the code in more detail.
+The edx-sysadmin plugin equips the admin users with certain handy operation in Open edX which would otherwise be difficult to perform manually.
+This plugin is just like other django applications except for the parts that really make it able to be integrated with edx-platform and to understand those you might want to take a look at `this link. <https://github.com/edx/edx-django-utils/tree/master/edx_django_utils/plugins>`_
```
Comment (ziafazal) [inline README.rst:16]: I don't think we need this paragraph

File: README.rst [hunk 3]
```diff
@@ -1,96 +1,90 @@
...
   git checkout -b <your_github_username>/<short_description>
-
   # Using your favorite editor, edit the code to make your change.
-  vim …
-
-  # Run your new tests
-  pytest ./path/to/new/tests
-
-  # Run all the tests and quality checks
-  make validate
```
Comment (ziafazal) [inline README.rst:68]: How we run tests in plugin? I mean there should be an alternate of `make validate` command to run unit and quality tests.

**Review by ziafazal — COMMENTED** [review]
@HamzaIbnFarooq see inline comments for question I have.

File: README.rst (hunk 1 above)
Comment (HamzaIbnFarooq) [reply README.rst:10]: It says `If you wish to use the Sysadmin Dashboard with Open edX releases before lilac you`. Only meant for old releases.

File: README.rst (hunk 3 above)
Comment (HamzaIbnFarooq) [reply README.rst:68]: As Arslan has created a new PR (https://github.com/mitodl/edx-sysadmin/pull/19) in which he changed the way tests will be run, I am currently looking into it but seems like on local dev the tests will only be run after integrating with edx.

Comment (HamzaIbnFarooq) [reply README.rst:68]: pylint and pytest both are dependent on edx modules.

File: README.rst (hunk 2 above)
Comment (HamzaIbnFarooq) [reply README.rst]: removed.

File: README.rst (hunk 1 above)
Comment (HamzaIbnFarooq) [reply README.rst]: I have updated the sentence now as it might had made a wrong sense before, kindly have a look now.

File: README.rst (hunk 3 above)
Comment (ziafazal) [reply README.rst:68]: We should then write some notes how we can install sys admin in development mode in edx-platform devstack and how we can run tests in lms-shell of devstack.

Comment (HamzaIbnFarooq) [reply README.rst:68]: I have added the integration testing details now.

**Review by ziafazal — APPROVED** [review]
👍

**Pull request merged**